### ISSUE TITLE
Stacking of DwarFS layers can be done in an OverlayFS manner

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/tebako#3]

**Comment by maxirmx** [comment]
DwarFS manuals are not applicable since they all relay on underling linux kernel.  Technically we are not implementing a filesystem, we will have a library that works one level up.

ruby packer / squashfs implement their own custom overlay functionality. It looks like this is the only possible approach as far as we want to have patched Ruby and not a solution integrated into operationg system kernel.

**Comment by ronaldtse** [comment]
@maxirmx exactly, that's why the issue says "OverlayFS manner" instead of using "OverlayFS"; the man page was given to illustrate the concept. It would need to be a user-space library that allows overlays.
